pecos/xmc/xlinear/model.py

```diff
@@ -57,6 +58,7 @@ class TrainParams(pecos.BaseParams):
         nr_splits: int = 2
         min_codes: int = None  # type: ignore
         shallow: bool = False
+        relevance_norm: str = "l1"
```

Comment:
as discussed in last meeting, should we use `rel_induce` and `rel_norm` to control the hierarchical bahavior of cost-sensitive learning?